### PR TITLE
fix(tile-converter): fix loading buffers in preprocess-3d-tiles

### PR DESCRIPTION
**Pull request by mspivak-actionengine**

### CODE REVIEW

**Review by ibgreen — APPROVED** [review]

**Review by belom88 — CHANGES_REQUESTED** [review]
This change doesn't fix the problem in `gltf`. The problem is that `KHR_texture_transform` extension fails if buffers are not loaded.
I suggest making a change in https://github.com/visgl/loaders.gl/blob/f51905b1385c8ba51c238adc70868fed997474fa/modules/gltf/src/lib/extensions/KHR_texture_transform.ts#L64
```
if (!hasExtension || !options.gltf?.loadBuffers) {
```
A test case in `modules/gltf/test/lib/extensions/KHR_texture_transform.spec.js` will be appreciated.

File: modules/gltf/src/lib/api/post-process-gltf.ts [hunk 1]
```diff
@@ -135,6 +135,12 @@ class GLTFPostProcessor {
     this.images = images;
     this.jsonUnprocessed = json;
 
+    if (!options?.loadBuffers) {
+      // @ts-expect-error
+      const jsonPost: GLTFPostprocessed = {...json};
```
Comment (belom88) [inline modules/gltf/src/lib/api/post-process-gltf.ts:140]: It doesn't return postprocessed gltf that is anoying for a user. I don't agree with this change

File: modules/gltf/test/lib/extensions/KHR_texture_transform.spec.js [hunk 2]
```diff
@@ -25,3 +25,22 @@ test('GLTFLoader#KHR_texture_transform', async (t) => {
   );
   t.end();
 });
+
+test('GLTFLoader#KHR_texture_transform with no buffers loaded', async (t) => {
+  const response = await fetchFile(GLTF_BINARY_URL);
+  const data = await response.arrayBuffer();
+  /**
+   * @type {import("@loaders.gl/gltf").ParseGLTFOptions}
+   */
```
Comment (belom88) [inline modules/gltf/test/lib/extensions/KHR_texture_transform.spec.js:34]: Convert the file to TypeScript probably

File: modules/gltf/test/lib/extensions/KHR_texture_transform.spec.js [hunk 3]
```diff
@@ -25,3 +25,22 @@ test('GLTFLoader#KHR_texture_transform', async (t) => {
...
+  const loaderOptions = {loadBuffers: false};
+  const gltfWithBuffers = await parse(data, GLTFLoader, {gltf: loaderOptions});
+  const gltf = postProcessGLTF(gltfWithBuffers, loaderOptions);
```
Comment (belom88) [inline modules/gltf/test/lib/extensions/KHR_texture_transform.spec.js:37]: We don't do postprocess gltf without buffer usually

**Review by belom88 — APPROVED** [review]

**Pull request merged**